apps/juicebox_web/assets/js/reactions/styles.scss

```diff
@@ -0,0 +1,21 @@
+.reactions {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+
+  video {
+    flex: 1 0 0px;
+    width: 0;
+    transition: all 0.3s;
+    &.new-reaction {
```

Comment:
Rule declaration should be preceded by an empty line

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#emptylinebetweenblocks)